### PR TITLE
MP-5711: Walkthrough guide for Marketplace Merchant feature

### PR DESCRIPTION
**Pull request by tobi812**
## PR Description

Added a Walkthrough guide for Marketplace Merchant feature.

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-feature-walkthrough.md [hunk 1]
```diff
@@ -14,6 +14,35 @@ To learn more about the feature and to find out how end users use it, see [Marke
...
+## Module dependency graph
+
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/42f7f5aa-2eee-4149-9b9e-50052e870946.png?utm_medium=live&utm_source=custom)
+
+
+| MODULE | DESCRIPTION |
+| -------------------- | ---------- |
+| [Merchant](https://github.com/spryker/merchant) | This module has DB structure and facade methods to save/update/remove Merchants. |
+| [MerchantProfile](https://github.com/spryker/merchant-profile) | MerchantProfile module provides data structure, facade methods and plugins for extending merchant by merchant profile data. |
+| [MerchantProfileDataImport](https://github.com/spryker/merchant-profile-data-import) | Importer for MerchantProfile data. |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-feature-walkthrough.md:26]:
```suggestion
| [MerchantProfileDataImport](https://github.com/spryker/merchant-profile-data-import) | Imports MerchantProfile data. |
```

**Review by ilyakubanov — COMMENTED** [review]
- Can we add something to description? It is too short 
-

File: docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-feature-walkthrough.md [hunk 2]
```diff
@@ -14,6 +14,35 @@ To learn more about the feature and to find out how end users use it, see [Marke
...
+| [MerchantProfileWidget](https://github.com/spryker-shop/merchant-profile-widget) | This module provides a merchant-profile molecule for for introducing merchant profile information. |
+| [MerchantWidget](https://github.com/spryker-shop/merchant-widget) | This module provides widget to display merchant information. |
+| [MerchantProfileMerchantPortalGui](https://github.com/spryker/merchant-profile-merchant-portal-gui) | This module provides Zed UI interface for merchant profile management for the Merchant portal. |
+| [MerchantRestApi](https://github.com/spryker/merchants-rest-api) | MerchantsRestApi module provides REST API endpoints to manage merchants. |
+| [MerchantRestApiExtension](https://github.com/spryker/merchants-rest-api-extension) | Provides plugin interfaces to extend MerchantsRestApi module from the other modules. |
+
+
+## Domain model
+
+![Domain Model](https://confluence-connect.gliffy.net/embed/image/73486462-e9d3-4eb2-93ef-a5cde49cce98.png?utm_medium=live&utm_source=custom)
```
[the remark this answers is not in the document]
Comment (zssamoylov) [reply docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-feature-walkthrough.md:43]: fixed

File: docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-feature-walkthrough.md [hunk 3]
```diff
@@ -14,6 +14,35 @@ To learn more about the feature and to find out how end users use it, see [Marke
 {% endinfo_block %}
 
 
+## Module dependency graph
+
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/42f7f5aa-2eee-4149-9b9e-50052e870946.png?utm_medium=live&utm_source=custom)
```
[the remark this answers is not in the document]
Comment (zssamoylov) [reply docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-feature-walkthrough.md:19]: fixed.

**Review by ilyakubanov — DISMISSED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**